README.md

```diff
@@ -1,9 +1,36 @@
-# 🚧 WORK IN PROGRESS. See [#1](https://github.com/oscard0m/octoherd-script-add-cache-to-node-github-action/pull/1) | [Preview](https://github.com/oscard0m/octoherd-script-add-cache-to-node-github-action/tree/initial-version)
-
 # octoherd-script-add-cache-to-node-github-action
 
 > Add cache parameter to GitHub Actions using setup-node
 
+[![@latest](https://img.shields.io/npm/v/octoherd-script-add-cache-to-node-github-action.svg)](https://www.npmjs.com/package/octoherd-script-add-cache-to-node-github-action)
+[![Build Status](https://github.com/oscard0m/octoherd-script-add-cache-to-node-github-action/workflows/Test/badge.svg)](https://github.com/oscard0m/octoherd-script-add-cache-to-node-github-action/actions?query=workflow%3ATest+branch%3Amain)
+
+## Usage
+
+Minimal usage
+
+```js
+npx octoherd-script-add-cache-to-node-github-action
+```
+
+Pass all options as CLI flags to avoid user prompts
+
+```js
+npx octoherd-script-add-cache-to-node-github-action \
+  -T ghp_0123456789abcdefghjklmnopqrstuvwxyzA \
+  -R "oscard0m/*" \
+  --cache "npm" | "yarn"
```

Comment:
I think I'd remove this line, the option is described in the table below, and passing `"npm" | "yarn"` is invalid. People will copy & paste this command and it will fail behave in away they don't expect, because the `|` will probably pipe all the output into the `yarn` binary

Reply:
> I think I'd remove this line, the option is described in the table below, and passing `"npm" | "yarn"` is invalid. People will copy & paste this command and it will fail behave in away they don't expect, because the `|` will probably pipe all the output into the `yarn` binary

super good catch, I fulfilled from create-octoherd-script command line and I did not review how it looked in the `README.md`